pyproject.toml

```diff
@@ -191,6 +191,7 @@ reportUnknownParameterType = false
 reportUnknownVariableType = false
 reportUnusedVariable = false
 reportPrivateUsage = false
+reportSelfClsParameterName = false
```

Comment:
Why did you add this?

Reply:
Oops I am sorry I changed it without leaving a comment, I forgot I made this change.

This is a lint rule, rather than a type check, that doesn’t apply to stubs. The name of the variable is out of control of the stub, it is defined by the implementation  (see for example [typeshed](https://github.com/python/typeshed/blob/8b9b0f4d031fe614a0211e1faadf57b3340e643e/pyrightconfig.json#L43-L44)). If, for instance, the implementation defines a method like:
```python
class C:
    @classmethod
    def method(klass): …
```
and the stub defines it as:
```python
class C:
    @classmethod
    def method(cls): …
```
just to silence the linter, then if someone calls `C.method(cls=…)` they will get a TypeError.

I started getting errors about this when I added the boxplot method.

Anyhow, if you don’t like this change please feel free to change it back. I won’t have access to computer for some time.